src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java

```diff
@@ -17,12 +23,14 @@
  */
 public class JavaFileWriter {
 
+    private List<JavaFile> javaFiles = new ArrayList<>();
+
     private final static Logger logger = LoggerFactory.getLogger(JavaFileWriter.class);
 
     /**
      * Calls a method to write each {@link JavaClass} in outputDir folder.
      *
-     * @param outputDir path to the folder where need to generate .java source files.
+     * @param outputDir   path to the folder where need to generate .java source files.
```

Comment:
no need in extra space

Reply:
These spaces were added by code formatter